### DISCUSSION TITLE
tictactoe example

### DISCUSSION THREAD

**Discussion by PedroBatistop** [neplexlabs/canvacord#99] [Q&A]
could you give me an example of how to use tictactoe, pls?

**Comment by twlite** [comment]
this library won't provide tictactoe game engine, you would need to implement the logic by yourself. And the tictactoe method provided by this lib is just for visualization.

**[The documentation](https://canvacord.js.org/docs/main/main/class/Canvacord?scrollTo=tictactoe)** says that `tictactoe` takes 2 parameters (both of them being object): `fill` and `color`.

`fill` is used to fill the cells of tictactoe board: `X` | `O` and `color` for obviously color formatting. (FYI: color is optional)

So the basic example would be:

```js
import { Canvas as canvacord } from "canvacord";

const game = getTicTacToe(); // implement this logic yourself
const cells = game.board.getCells(); // implement this logic yourself, imagine it returns object { a1: "X", b2: "O", ... }
const board = canvacord.tictactoe(cells); // image buffer, do something with this
```
